spec/support/helpers/state_file_intake_helper.rb

```diff
@@ -45,9 +45,6 @@ def step_through_eligibility_screener(us_state:)
       expect(page).to have_text I18n.t("state_file.questions.eligible.id_credits_unsupported.dependents_not_claimed_fed_return")
       expect(page).to have_text I18n.t("state_file.questions.eligible.id_credits_unsupported.change_in_filing_status")
     when "md"
-      expect(page).to have_text I18n.t("state_file.questions.md_eligibility_filing_status.edit.title", year: filing_year)
-      click_on I18n.t("general.continue")
-
```

Comment:
this fixes the `review_page_spec` MD specs failing when run locally (@mrotondo discovered this). This step is checking whether clicking on the eligibility filing status page without choosing an option will re-render the page. Why it fails locally is still a mystery, but it didn't feel like it should be a part of the `step_through_eligibility_screener` which gets used all over the specs. Rather this kind of test should move into the controller, if not already present.

Reply:
addedd spec/forms/state_file/md_eligibility_filing_status_form_spec.rb